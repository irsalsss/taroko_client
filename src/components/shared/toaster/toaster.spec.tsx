import { render } from "@/utils/test/react-testing-setup";
import Toaster from "./toaster";

describe("Toaster", () => {
  it("should render toaster successfully", () => {
    const { baseElement } = render(<Toaster />);

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
