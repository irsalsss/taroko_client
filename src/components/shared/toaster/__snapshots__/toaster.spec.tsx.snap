// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toaster should render toaster successfully 1`] = `
<body>
  <div>
    <div
      class="Toastify"
    />
  </div>
</body>
`;
